### ISSUE TITLE
Error response from daemon: i/o timeout

### ISSUE THREAD

**Issue by kstenerud** [docker/for-win#4710]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: C39B323E-C0E4-460E-BC40-4C119A2EA8F5/20190913161637

### Expected behavior

It lists the docker images

### Actual behavior

It prints "Error response from daemon: i/o timeout"

### Information

This is my first time installing docker deskdop. I installed it on a fresh windows 10 vm.
This is 100% reproducible. Docker simply does not work.

  - Windows Version: Windows 10 (1903) running under kvm

```
PS C:\Users\karl> docker version
Client: Docker Engine - Community
 Version:           19.03.2
 API version:       1.40
 Go version:        go1.12.8
 Git commit:        6a30dfc
 Built:             Thu Aug 29 05:26:49 2019
 OS/Arch:           windows/amd64
 Experimental:      true
Error response from daemon: Bad response from Docker engine
```

### Steps to reproduce the behavior

  1. Type "docker images"

**Comment by zer0def** [comment]
**Presumably** you're running Docker Desktop for Windows from the stable channel, in which case this requires explicit statement (ideally a dedicated section on the downloads page in huge red blinking marquee) to put the virtualization nesting issue to rest until the kernel major version gets bumped in stable:
- based on linuxkit/linuxkit#3209 the reason why Docker's VM does not start in nested virtualization environments is because LTS kernels prior to 4.14 are unable to properly synchronize their clock sources, which results in the VM getting restarted infinitely; this behavior is specific to running Hyper-V as a second level hypervisor and you can confirm this by checking out the boot logs in `C:\Users\%USERNAME%\AppData\Local\Docker\log`
- based on [Docker Desktop for Windows Edge release notes](https://docs.docker.com/docker-for-windows/edge-release-notes/), version 2.1.1.0 released in August 2019 accidentally addresses this issue due to a kernel upgrade (tested it on HyperV-in-KVM just prior to this post)

TL;DR: Install Docker Desktop for Windows from the edge channel until someone finally decides to make an equivalent upgrade in stable.
